src/NexusMods.App.UI/DiagnosticSystem/LoadoutItemGroupFormatter.cs

```diff
@@ -11,7 +11,14 @@ internal sealed class LoadoutItemGroupFormatter(IConnection conn) : IValueFormat
     public void Format(IDiagnosticWriter writer, ref DiagnosticWriterState state, LoadoutItemGroupReference value)
     {
         // TODO: custom markdown control
-        var mod = LoadoutItemGroup.Load(conn.Db, value.DataId);
-        writer.Write(ref state, $"{mod.AsLoadoutItem().Name}");
+        var loadoutItemGroup = LoadoutItemGroup.Load(conn.Db, value.DataId);
```

Comment:
You sure Load doesn't just throw exception?

Reply:
The `Load` method just calls the constructor of the struct. It doesn't do anything, it just sets properties.